src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+@font-face {
+  font-family: Roboto;
+  src: url(Roboto/Roboto-Medium.ttf);
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+body {
+  margin: 0;
+}
+
+ul {
+  padding-left: 0;
+  margin: 0;
+}
+
+.header {
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  position: relative;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+}
+
+.nav {
+  --hover-color: #00acdc;
```

Comment:
move to `::root`

Reply:
@etojeDenys 
changed, but 2 passed remained the same